### PR TITLE
Add managment IP validation

### PR DESCRIPTION
**Pull request by szok**
Added validation to managment asset IP

### CODE REVIEW

File: src/ralph/data_center/models/networks.py [hunk 1]
```diff
@@ -529,15 +514,7 @@ def save(self, allow_device_change=True, *args, **kwargs):
         if settings.CHECK_IP_HOSTNAME_ON_SAVE:
             if not self.address:
                 self.address = network.hostname(self.hostname, reverse=True)
-            if not self.hostname:
-                self.hostname = network.hostname(self.address)
         self.number = int(ipaddress.ip_address(self.address))
-        try:
-            self.network = Network.from_ip(self.address)
```
Comment (mkurek) [inline src/ralph/data_center/models/networks.py:536]: remove this method from Network model too - if we'll need this in the future we'll copy it from Ralph 2.0 - for now it's useless and unused code.

File: src/ralph/data_center/forms/network.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        is_management = []
+        for form in self.forms:
+            cleaned_data = form.cleaned_data
+            if (
+                cleaned_data and
+                not cleaned_data.get('DELETE', False)
+            ):
+                is_management.append(cleaned_data.get('is_management'))
+
+        if is_management.count(True) > 1:
```
Comment (mkurek) [inline src/ralph/data_center/forms/network.py]: `if is_management and is_management.count(True) != 1` - if there is any ip, one should be marked as management

**Comment by ghost** [comment]
:ok:

File: src/ralph/data_center/forms/network.py (hunk 2 above)
Comment (ghost) [inline src/ralph/data_center/forms/network.py]: `count` on empty list also works so:
`if  is_management.count(True) != 1`
is enough, what do you think?

File: src/ralph/data_center/tests/test_forms.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        with self.assertRaisesRegexp(
+            ValidationError,
+            (
+                'Only one managment IP address can be assigned '
+                'to this asset'
+            )
+        ):
+            validate_is_management([form_1, form_2])
+
+    def test_vlaidate_not_management(self):
```
Comment (mkurek) [inline src/ralph/data_center/tests/test_forms.py]: validate

**Comment by mkurek** [comment]
:ok:

**Pull request merged**